invenio_saml/handlers.py

```diff
@@ -40,6 +40,12 @@ def default_account_info(attributes, remote_app):
 
     mappings = remote_app_config["mappings"]
 
+    affiliations_mapping = mappings.get("affiliations", None)
+    if affiliations_mapping:
+        affilliations = attributes[affiliations_mapping]
```

Comment:
minor: typo `affilliations` -> `affiliations` (one `l` to much)

Reply:
Great catch, fixed!